File: tests/benchmark/fib/fib.js
function fib(n) {
	if (n < 2) return n;
	return fib(n - 1) + fib(n - 2);
}

var start = process.hrtime();
for (var i=0; i < 10; i++) {
	process.stdout.write(fib(30) + '\n');
}

var end = process.hrtime(start);
var secs = (end[0] + end[1] / 1e9).toFixed(6) + 's';
console.log('elapsed:', secs);

File: tests/benchmarks/fib/fib.lua
local function fib(n)
	if n < 2 then
		return n
	else
		return fib(n - 1) + fib(n - 2)
	end
end

local start = os.clock()

for i=1, 10 do
	print(fib(30))
end

local seconds = os.clock() - start
print('elapsed: ' .. seconds .. 's')
